### DISCUSSION TITLE
Depreciation calculation

### DISCUSSION THREAD

**Discussion by jainjine** [snipe/snipe-it#10007] [General]
Hi There,

I just want to check the depreciation calculations. How does it working and what is the taken percentage as taking for the diff.value ?. 
Because I think the depreciation percentage is different for the each country right? so is there any manual percentage insertion option ?

For example : 
my asset total cost is INR 68000 and  the depreciation would be 25% or something for 12 months so for this how should I set  in SnipeIT?

Thanks
Jineesh

**Comment by thanukalpa** [comment]
unsubscribe

**Reply by jainjine** [reply]
Did meaned  the depreciation does works in paid version?

**Comment by thanukalpa** [comment]
unsubscribe

Kalpa De Silva
Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

From: ***@***.***>
Sent: Monday, August 30, 2021 4:57 PM
To: ***@***.***>
Cc: ***@***.***>
Subject: [snipe/snipe-it] Depreciation calculation (#10007)


Hi There,

I just want to check the depreciation calculations. How does it working and what is the taken percentage as taking for the diff.value ?.
Because I think the depreciation percentage is different for the each country right? so is there any manual percentage insertion option ?

For example :
my asset total cost is INR 68000 and the depreciation would be 25% or something for 12 months so for this how should I set in SnipeIT?

Thanks
Jineesh

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/snipe/snipe-it/discussions/10007>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AIVHGZ3W2UI2ILZ4GNXY7CLT7NTLTANCNFSM5DBVLKVA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
